avm/res/document-db/database-account/main.bicep

```diff
@@ -203,15 +203,15 @@ var defaultFailoverLocation = [
   {
     failoverPriority: 0
     locationName: location
-    isZoneRedundant: true
```

Comment:
Per AVM guidelines zone redundant needs to be always true

Reply:
Just FYI https://azure.github.io/Azure-Verified-Modules/faq/#what-does-avm-mean-by-waf-aligned

Reply:
Unfortunately, the product team suggested against this. Capacity for zone-redundant accounts is constrained in some PAYG Azure subscriptions and other subscriptions we use for internal testing.

Some of the pipeline error messages directly call out the fact that the target subscription doesn't have capacity for a zonal account.

So, I'm leaning very heavily on the *[..], where possible and appropriate [...]* line from that WAF-aligned standard. I would argue here that zone redundancy may not be possible or appropriate as a default based on current capacity.

@markjbrown, can you provide some more detail here.

Reply:
Let's leave this review open while we discuss current product capacity and the WAF-aligned standard here: https://github.com/Azure/bicep-registry-modules/issues/3588#issuecomment-2428562666

Reply:
Per https://github.com/Azure/bicep-registry-modules/issues/3588#issuecomment-2429673345, they do want this change reverted. Unfortunately, this means that we will need to accept some possibility that the e2e and multi-region tests fail due to capacity.

I reverted this change and this should resolve the current conversation.